website/src/documentation/sdks/java-dependencies.md

```diff
@@ -26,336 +26,12 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+To see the compile and runtime dependencies for your Beam SDK version, dependencies are listed in `https://raw.githubusercontent.com/apache/beam/v<version_number>/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy` under `project.ext.library`. 
```

Comment:
WDYT about turning this into a stepped list, with explicit instructions for replacing "<version_number>" with the major.minor.revision string? Just concerned folks won't know to specify the full version number (e.g., "2.16" returns a 404)

Reply:
That's an excellent point--I'd explain the placeholder and format it differently 

https://developers.google.com/style/code-in-text#explaining-placeholders

Reply:
I'd also consider using all-uppercase characters here instead of the all-lowercase recommendation in the Google style guide. Since this placeholder is just in plain text (no bold/italics/pink color), the all-uppercase characters will help distinguish it as a variable.

Reply:
It looks like we might want to rephrase the first sentence. Should we say "Compile and runtime dependencies for your Beam SDK version dependencies are listed in ..."

Reply:
I addressed the feedback, but left the placeholder as lowercase per the Google style guide. Given the additional example and note below the URL, I *think* it will be clear to users what the placeholder is and what to replace it with.